unordered_map/emplace_hint.md

```diff
@@ -0,0 +1,32 @@
+# emplace_hint
+**Description :**  The function accepts three mandatory parameters i.e position, key, element in similar order. Where 'key' specifies the key to be inserted in the unordered_map container. 'element' specifies the element to the key which is to be inserted in the map container. 'position' specifies the position from where the search operation for the ordering is to be started, hence making the insertion faster. The function does not return anything. It works similarly to unordered_map emplace() function but if the user provides position accurately than it works faster then it
```

Comment:
The wording in description can be improved.
The lines: key specifies........, element specifies ...... position specifies .....
All of these linse seems repeated and verbose, remember the goal is to read this in 30 seconds.